replica/database.cc

```diff
@@ -393,6 +395,24 @@ database::database(const db::config& cfg, database_config dbcfg, service::migrat
     if (_dbcfg.sstables_format) {
         set_format(*_dbcfg.sstables_format);
     }
+
+    _tp_timer.arm_periodic(std::chrono::seconds(_cfg.persistent_toppartitions_publish_interval_sec()));
+}
+
+void database::on_tp_timer() {
+    size_t list_size = _cfg.persistent_toppartitions_list_size();
+
+    for (auto& d: _tp_listener->get_top_k_read().top(list_size).values) {
+        auto partition = ("(" + d.item.schema->ks_name() + ":" + d.item.schema->cf_name() + ") ") + sstring(d.item);
+        metricslog.debug("Toppartitions - Read - Partition [{}]: {}", partition, d.count);
```

Comment:
If we are doing all this work of gathering the metrics and generating a report, we might as well use `info` so it is actually appears in the logs.

Reply:
Done.